README.md

```diff
@@ -58,3 +60,7 @@ scripts/e vs_devenv
 ## How do I pull the upstream changes from Electron?
 
 Run `scripts/pull-upstream-changes.sh`
+
+## How do I update LibreOfficeKit after compiling it locally?
+
+Run `scripts/local-lok.sh`
```

Comment:
The name of the script is a little vague out of the context of the README. Maybe `update-local-lok.sh`?